### ISSUE TITLE
cannot find function `SQLConnect` in this scope

### ISSUE THREAD

**Issue by brandonros** [pacman82/odbc-safe#7]
```
error[E0425]: cannot find function `SQLConnect` in this scope
  --> /Users/brandonros/.cargo/registry/src/github.com-1ecc6299db9ec823/odbc-safe-0.4.1/src/handles/hdbc.rs:87:13
   |
87 |             SQLConnect(
   |             ^^^^^^^^^^ did you mean `SQLConnectW`?

error: aborting due to previous error

For more information about this error, try `rustc --explain E0425`.
error: Could not compile `odbc-safe`.

```

**Comment by brandonros** [comment]
I would paste a code sample... but... I don't think it's trying to compile any of my code? It looks like it is just trying to compile the library on its own?

Unless I am just failing to link my driver/library properly?

**Comment by brandonros** [comment]
https://github.com/pacman82/odbc-sys/pull/18

**Comment by adamreichold** [comment]
@brandonros Can this be closed after pacman82/odbc-sys#18 was merged?
